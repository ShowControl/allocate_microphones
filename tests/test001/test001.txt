[show test001]  // one actor, one character
Designer=John Sauter
DateNumeric=2019-09-15
DateText=September 16, 2019

[character character_01]

[actor actor_01]

[role]
actor=actor_01
character=character_01

[page 1]
[act 1]
[scene 1]
[enter character_01]
[page 2]
[exit character_01]

[end show]

