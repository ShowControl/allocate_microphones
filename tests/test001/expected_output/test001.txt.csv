page, "actor_01"
"1","*"
"2"," "
